velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -94,6 +94,43 @@ RowVectorPtr reduceToSelectedRows(
   reducedVector->copy(rowVector.get(), rowsToCopy, rawIndices);
   return reducedVector;
 }
+
+// Returns the expression set used by common expression eval path.
+exec::ExprSet createExprSetCommon(
+    const std::vector<core::TypedExprPtr>& plans,
+    core::ExecCtx* execCtx,
+    bool disableConstantFolding) {
+  std::shared_ptr<exec::ExprSet> exprSetCommon;
```

Comment:
This line can be removed now.

Reply:
@rui-mo could you address this so I can merge (I'm eager to see this fixed)